airflow/jobs.py

```diff
@@ -1231,6 +1232,17 @@ def _log_file_processing_stats(self,
                        if processor_start_time else None)
             last_run = processor_manager.get_last_finish_time(file_path)
 
+            file_name = file_path[len(dags_folder) + 1:]
+            name = os.path.splitext(file_name)[0].replace(os.sep, '.')
+
+            if last_runtime is not None:
+                Stats.gauge('last_runtime.{}'.format(name), last_runtime)
```

Comment:
Interesting choice to use epoch as a gauge, it should work well! I dug a bit following the `processor_manager` and was surprised to see how much this part of the code has changed since I first wrote it.